app/views/transacs/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: @transac, url: category_transacs_path(@category), class: 'new_transac') do |form| %>
+  <% if transac.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(transac.errors.count, "error") %> prohibited this transac from being saved:</h2>
+
+      <ul>
+        <% transac.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class ='field'>
+    <%= form.text_field :name %>
+  </div>
+
+  <div class ='field'>
+    <%= form.text_field :amount %>
+  </div>
+
+  <div class ='field'>
+    <%= form.collection_select :category_id, @categories, :id, :name, required: true %>
+  </div>
```

Comment:
- [ ] As mentioned by the previous reviewer in this [PR](https://github.com/SanaTameem/budget-buddy-old/pull/10) :point_left: When creating a new transaction, at least one category should be selected for the transaction. I can see that you have the category present, but a user can create a transaction without adding a type as a parameter. Kindly add some validations so that the user is asked to specify the type when making a transaction but still you didn't fix this issue kindly fix it :slightly_smiling_face: 

_**Note: Please don't create a new repo for a new PR make a change and commit your code in the same repo don't create a new repo for new PR**_

![Screenshot from 2023-11-04 22-34-42](https://github.com/SanaTameem/budget-buddy/assets/45226848/2da8ae01-cee7-489b-9a6b-b9fa1f26da4d)

Reply:
Hello @aamirkhan2478 
Sir, Could you please provide me with a little more details? How should I do it?